lib/acpi/Kconfig

```diff
@@ -30,6 +30,10 @@ config ACPI_MAX_PRT_ENTRY
 
 endif # PCIE_PRT
 
+# Default minimum system heap required by ACPI
+config HEAP_MEM_POOL_ADD_SIZE_ACPI
+	def_int 1048576 # 1MB
```

Comment:
Nothing to do here but in future we could have a template for this similar to the logging one